### ISSUE TITLE
Feature request: simplify the start page

### ISSUE THREAD

**Issue by sean-kang** [zanllp/sd-webui-infinite-image-browsing#93]
Launch from Quick Move in the start page looks overcomplicated with items that are never used. For example, I see the following items in my start page.
```
    Text to image
    Image to image
    Directory for saving images using the Save button
    Extra
    Image to image grid
    Text to image grid
    hypernetworks
    Embedding
    Working folder
    home
```
There are too many items!

1. Browsing image grids is sort of useless because this extension does better job at viewing lots of images. They can be removed.
2. I don't even know why there are the model directories such as hypbernetwork and Embedding are here. Those are not image directories. They can be removed.
3. working folder and home are not frequently unless the extension is used as a generic image browser. They can be removed?
4. wordings are inconsistent with the terms used in Stable Diffusion Web UI.

TLDR I think the list can be simplified like this.
```
   txt2img
   img2img
   extra
   save
```

**Comment by zanllp** [comment]
I will simplify the names and remove 'Hypernetworks' and 'Embedding'. These names were used previously, as the extension had some file browser functionalities, which have now been greatly simplified.

'Working folder' and 'home' buttons will continue to exist, as the extension supports independent operation (https://github.com/zanllp/sd-webui-infinite-image-browsing/issues/47). Having these two buttons will be convenient when running independently without passing any parameters.

**Comment by zanllp** [comment]
I have implemented this, but I kept the grid as I feel that some people might still find it useful.
so I have closed this issue. If you have any further problems or requests, please create a new issue.
